### PR TITLE
test: fix readline dependent tests

### LINKED ISSUE #7620: tests are failed on dev installation with custom .inputrc config
Namely test gh_5064. By the way the test has tweak to handle same issue for specific .inputrc on Centos 7.

```
-- Readline on CentOS 7 produces \e[?1034h escape sequence before tarantool> prompt, remove it.
local result = fh:read('*a'):gsub('\x1b%[%?1034h', '')
```

One more test failing for same reason is gh_2717.

### PR DESCRIPTION
**Pull request by nshy**
The tests fail on dev installation if it's readline configuration
(.inputrc) changes prompt. Use default readline configuration for
the tests.

Closes #7620

NO_DOC=test changes
NO_TEST=test changes
NO_CHANGELOG=test changes

### CODE REVIEW

File: test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua [hunk 1]
```diff
@@ -11,12 +11,14 @@ tarantool> ]]
...
 
 g.test_console_ignores_i_flag_without_tty = function()
-    local cmd = [[printf '42\n' | ]] .. TARANTOOL_PATH .. [[ -i 2>/dev/null]]
-    local fh = io.popen(cmd, 'r')
 
-    -- Readline on CentOS 7 produces \e[?1034h escape sequence before tarantool> prompt, remove it.
-    local result = fh:read('*a'):gsub('\x1b%[%?1034h', '')
+    -- Suppress prompt changes by readline by setting nonexistent inputrc file
+    -- to simplify assertions.
+    local cmd = [[printf '42\n' | ]] .. 'INPUTRC=Pheiphe2 ' ..
```
[the remark this answers is not in the document]
Comment (Lord-KA) [reply test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua:17]: I agree with Sergey, this way you can shorten comment and save readability. The "other places" are all in [gh-2717-no-quit-sigint.test.lua](https://github.com/tarantool/tarantool/pull/7621/commits/976b3bc22a59f649d17864713c32e4bac9b99ed2#diff-321025ff9a1bd62f89bdefd09fe95897dbdb3cc6cf258625440120004d409a74) and don't have explicit comments about the meaning of the string. One could confuse them with a weird readline configuration name.

**Review by Lord-KA — APPROVED** [review]
I could not find a better way to set Readline configuration to default, so this seems fine, except for the comment above. LGTM after fixing it.

File: test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (Totktonada) [reply test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua:17]: We recently looked at the relevant tests with @tsafin and initially we both were confused by this string. So +2 here.

[the remark this answers is not in the document]
Comment (nshy) [reply test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua:17]: ok

[the remark this answers is not in the document]
Comment (Lord-KA) [reply test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua:17]: The alternative to `non_existent_inputrc` could be `/dev/null`.

**Comment by nshy** [comment]
Centos 7 still fails if we remove that gsub deleting \e[?1034h escape sequence. So looks like it is compiled in. By the way `gh-2717-no-quit-sigint.test.lua` that makes similar tests just disabled.

I'll return gsub back.

**Review by Lord-KA — APPROVED** [review]
Thanks for the patch, LGTM.

File: test/app-tap/gh-2717-no-quit-sigint.test.lua [hunk 2]
```diff
@@ -19,7 +19,7 @@ local TARANTOOL_PATH = arg[-1]
 local test = tap.test('gh-2717-no-quit-sigint')
 
 test:plan(8)
-local cmd = 'INPUTRC=Pheiphe2 ' .. TARANTOOL_PATH .. ' -i 2>&1'
```
Comment (locker) [inline test/app-tap/gh-2717-no-quit-sigint.test.lua:22]: Why is `Pheiphe2` any better than `non_existent_file`? Why isn't another test updated? IMO better create a real inputrc file for tests.

Anyway, the two patches should be squashed.

**Review by locker — CHANGES_REQUESTED** [review]

File: test/app-tap/gh-2717-no-quit-sigint.test.lua (hunk 2 above)
Comment (locker) [reply test/app-tap/gh-2717-no-quit-sigint.test.lua:22]: Sorry - thought you replaced `non_existent_file` with `Pheiphei2`. Never mind - everything's fine. Still, I wouldn't split patches in such a fine grained way - "While we are at it, fix the non existent file name" in the commit message would do just fine.

**Review by locker — APPROVED** [review]

**Pull request merged**